frontend/components/Domain/Recipe/RecipeCardSection.vue

```diff
@@ -5,7 +5,7 @@
         <v-icon v-if="title" large left>
           {{ displayTitleIcon }}
         </v-icon>
-        <v-toolbar-title class="headline"> {{ title }} </v-toolbar-title>
+        <v-toolbar-title class="headline"> {{ title }} ({{ recipes.length }})</v-toolbar-title>
```

Comment:
Won't this only show the currently fetched recipes (i.e. _after_ pagination)? We return the true "total" with the pagination API, should we use that value instead?

Reply:
This is what the API returns: https://github.com/mealie-recipes/mealie/pull/4489

Reply:
Ah, I see. I hadn't encountered any pagination yet in my usage.  I'll have a look and take another pass

Reply:
IIRC the frontend pulls two pages at once (with a page size of 32 that's 64 recipes) so you might want to add 100 recipes or so to your dev instance